modules/data.atmosphere/R/download.NARR_site.R

```diff
@@ -236,8 +236,8 @@ get_NARR_thredds <- function(start_date, end_date, lat.in, lon.in,
       ),
         robustly(get_narr_url)(url, xy = xy, flx = flx)
```

Comment:
This line now fails with a global variable warning -- It's the same problem as in dplyr functions (non-standard evaluation leading to the variable name being defined somewhere R's scope analysis doesn't know about), but we can't fix it by changing to `.data$flx` because foreach doesn't know how to use the rlang `.data` adverb. The two easiest options I see:

a. Re-add the line `get_NARR_thredds: no visible binding for global variable 'flx'` to `Rcheck_reference.log` and therefore keep ignoring the message
b. Add `flx <- NULL` between current lines 230 and 231, so that R can see it's defined but it doesn't interfere with the definitions we actually use (which are provided by `foreach`)

I'd slightly favor the NULL approach, but I'll defer to @ashiklom's opinion here if he has one.

Reply:
@infotroph just out of curiosity , how did you find the specific spot i.e. between line 230 and 231 for ' flx <- NULL ' to be added?

Reply:
@moki1202 I looked for the closest place that immediately before the problematic code but wasn't in the middle of another function or flow control operation, and since lines 232-238 were all part of the same variable assignment I decided to put it on the line before that. 

The other option would be to put it at the very top of the file, with a comment explaining what it does. That would make sense if we had a file with a lot of `foreach` calls in it that used the same variables over and over, but for the case where there's only one line complaining I think it makes sense to keep the fix as close to the issue as possible.

Reply:
@infotroph thank you for the clarification.